<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>message</class>
 <widget class="QWidget" name="message">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
	</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTableWidget" name="tableWidget">
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Arial&quot;;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="1" column="0">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="idLineEdit">
       <property name="minimumSize">
        <size>
         <width>35</width>
         <height>35</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Arial&quot;;</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QPushButton" name="deletemess">
         <property name="styleSheet">
          <string notr="true">font: 12pt &quot;Arial&quot;;</string>
         </property>
         <property name="text">
          <string>删除信息</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="backButton">
         <property name="styleSheet">
          <string notr="true">font: 12pt &quot;Arial&quot;;</string>
         </property>
         <property name="text">
          <string>返回</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Arial&quot;;</string>
       </property>
       <property name="text">
        <string>输入发送人学号</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="lookmessage">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Arial&quot;;</string>
       </property>
       <property name="text">
        <string>查看信息</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="seeAllButton">
       <property name="text">
        <string>刷新</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
